vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -26,21 +26,31 @@
         </div>
       </div>
     </div>
-    <div class="tw-relative tw-mt-[3rem]">
+    <div
+      class="tw-relative tw-mt-[3rem]"
+      :class="width < 768 || width > 2000 ? 'tw-container' : ''"
+    >
       <img
         v-if="width < 680"
         :src="bgMobile"
         loading="lazy"
         class="md:tw-hidden tw-absolute tw-top-0 -tw-right-10 sm:-tw-right-20 tw-h-full"
         alt="blog-background-image"
       />
+      <img
+        v-else
+        :src="bg"
+        loading="lazy"
+        class="tw-hidden lg:tw-block tw-absolute tw-top-0 tw-right-0"
+        alt="blog-background-image"
+      />
     </div>
     <div
-      class="tw-flex tw-flex-col md:tw-flex-row md:tw-mt-[4rem] tw-z-[1]"
+      class="tw-flex tw-flex-col md:tw-flex-row md:tw-mt-[4rem] tw-z-[1] lg:tw-h-[580px] md:tw-py-[10%] lg:tw-py-0"
```

Comment:
We can't use manual height right?

Reply:
Not in all cases, but in extreme cases.
Like in the blogs section, the image is absolutely positioned and depends on this div's height. So it's cutting when we have 1 or 2 blogs. To fix this, I have used the image's real height which is 580px